packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -45,7 +45,7 @@ const useStyles = makeStyles({
 /**
  * Applies style classnames to slots
  */
-export const usePresenceBadgeStyles = (state: PresenceBadgeState) => {
+export const usePresenceBadgeStyles: (state: PresenceBadgeState) => BadgeState = state => {
```

Comment:
This hook should return `PresenceBadgeState`:

```suggestion
export const usePresenceBadgeStyles = (state: PresenceBadgeState): PresenceBadgeState => {
```

➕

```diff
    state.outOfOffice && (state.status === 'busy' || state.status === 'doNotDisturb') && styles.outOfOfficeBusy,
    state.outOfOffice && state.status === 'away' && styles.outOfOfficeAway,
    state.className,
  );
- return useBadgeStyles(state);
+ return useBadgeStyles(state) as PresenceBadgeState;
```